draft-ietf-quic-qpack.md

```diff
@@ -1004,6 +1002,119 @@ Description:
 
 --- back
 
+# Static Table
+
+| Index | Name                             | Value                                                 |
+| ----- | -------------------------------- | ----------------------------------------------------- |
+| 0     | :authority                       |                                                       |
+| 1     | :path                            | /                                                     |
+| 2     | age                              | 0                                                     |
```

Comment:
Seems pretty low-value; will never appear in a request, and it's just three letters.